sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -1,477 +1,313 @@
-WITH _metrics_ping_distinct_client_count AS (
+WITH baseline_clients AS (
   SELECT
-    DATE(submission_timestamp) AS submission_date,
-    COUNT(DISTINCT client_info.client_id) AS metrics_ping_distinct_client_count
+    ping_date,
+    client_id,
+    channel,
+    country
   FROM
-    firefox_ios.metrics
+    (
```

Comment:
We should consider using `QUALIFY` instead of the subquery for the deduplication for better readability.